<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <arg name="topics" default="['/image', '/tf', '/point_cloud2']"/>
    <arg name="hdf5FolderPath" default="/seerep/seerep-data/"/>
    <arg name="project_frame_id" default="map"/>
    <arg name="project_name" default="testproject"/>
    <node name="seerep_ros_communication_hdf5dump" pkg="seerep_ros_communication" type="seerep_ros_communication_hdf5dump" output="screen">
        <rosparam param="topics" subst_value="True">$(arg topics)</rosparam>
        <rosparam param="hdf5FolderPath" subst_value="True">$(arg hdf5FolderPath)</rosparam>
        <rosparam param="project_frame_id" subst_value="True">$(arg project_frame_id)</rosparam>
        <rosparam param="project_name" subst_value="True">$(arg project_name)</rosparam>
    </node>
</launch>
